### PR TITLE
[PLTF-693] Upgrade to V0.5.0.rc1

### PR DESCRIPTION
**Pull request by robert-hromej**
[PLTF-693](https://opengovinc.atlassian.net/browse/PLTF-693)

Merge into from upstream https://github.com/ActsAsParanoid/acts_as_paranoid/tree/v0.5.0.rc1 for supporting rails5 versions.
Nothing else has changed in this PR.

### CODE REVIEW

**Review by nikita-andreev — APPROVED** [review]
LGTM

File: .travis.yml [hunk 1]
```diff
@@ -1,23 +1,39 @@
...
   - 1.9.3
   - 2.0.0
-  - 2.1
+  - 2.1.9
+  - 2.2.5
+  - 2.3.1
   - ruby-head
   - jruby
-  - jruby-9.1.7.0
+  - jruby-9.1.0.0
```
Comment (cademarkegard) [inline .travis.yml:17]: why are we downgrading here?

File: .travis.yml [hunk 2]
```diff
@@ -1,23 +1,39 @@
 language: ruby
+
+before_install:
+  - rvm get head
+  - gem install bundler
+
 script: "bundle exec rake test"
 sudo: required
 rvm:
```
Comment (cademarkegard) [inline .travis.yml:9]: Are these the ruby versions? If so, why aren't we including supported versions of ruby?

**Comment by cademarkegard** [comment]
are these changes just what is coming from upstream? If so can you please update the PR description?

File: .travis.yml (hunk 2 above)
Comment (robert-hromej) [reply .travis.yml:9]: > are these changes just what is coming from upstream? If so can you please update the PR description?

Yes, All changes from upstream(https://github.com/ActsAsParanoid/acts_as_paranoid/tree/v0.5.0.rc1).

**Comment by robert-hromej** [comment]
> are these changes just what is coming from upstream? If so can you please update the PR description?

Yes, All changes from upstream(https://github.com/ActsAsParanoid/acts_as_paranoid/tree/v0.5.0.rc1).
Description is updated

**Comment by robert-hromej** [comment]
> [PLTF-693](https://opengovinc.atlassian.net/browse/PLTF-693)

Merge into from upstream https://github.com/ActsAsParanoid/acts_as_paranoid/tree/v0.5.0.rc1 for supporting rails5 versions.
Nothing else has changed in this PR.

**Pull request merged**